e2e/tests/app.e2e.js

```diff
@@ -0,0 +1,33 @@
+import { test, expect } from '@playwright/test';
```

Comment:
Nit: Order imported symbol names alphabetically:

```suggestion
import { expect, test } from '@playwright/test';
```

(Can we configure ESLint to complain about unordered imports (both paths and symbol names)?)

Reply:
We can, there is a [sort-imports](https://eslint.org/docs/latest/rules/sort-imports) eslint rule that will enable reporting of such errors. The only question is how we want to configure it, because the possibilities are many. 
Please check [angularjs-angular-hybrid-angular-cli eslint configuration](https://github.com/xlts-dev/angularjs-angular-hybrid-angular-cli/pull/2/files#diff-6884918dc8291219be508e05e28965b958c734def85324f3b53858ea4702090fR120)

In my opinion, it is a bit annoying to receive an error because of unsorted imports ;)

Reply:
Changed

Reply:
What is annoying, is getting comments from me on PR reviews about unsorted imports 😛
I generally like alphabetically sorted stuff (mainly because it makes it easier to locate things), but happy to not enforce it with linting.